assets/queries/k8s/app_armor_config/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "app_armor_config",
+  "queryName": "app_armor_config",
+  "severity": "LOW",
+  "category": null,
+  "descriptionText": "Containers should be configured with AppArmor for any application to reduce its potential attack",
+  "descriptionUrl": "https://gsl.dome9.com/D9.K8S.IAM.23.html"
```

Comment:
The URL should point to official kubernetes documentation. Use this one "https://kubernetes.io/docs/tutorials/clusters/apparmor/".